.storybook/helpers/getArgTypes.ts

```diff
@@ -41,7 +41,7 @@ export function getArgTypes(component, options = { variables: {}}) {
     return name
   }
   const getTable = (name, defaultValue) => {
-    const value = defaultValue.value.replace(/'/gm, '').replace(/"/gm, '')
+    const value = defaultValue.value.replace(/"/gm, '')
```

Comment:
Why do we no longer need to replace `'`?

Reply:
It's a docs related issue. We need it to fix default values in Storybook controls.